util/llpcElfWriter.cpp

```diff
@@ -631,17 +631,9 @@ void ElfWriter<Elf>::WriteToBuffer(
 
 // =====================================================================================================================
 template<class Elf>
-Result ElfWriter<Elf>::ReadFromBuffer(
-    const void* pBuffer,    // [in] Buffer to read data from
-    size_t      bufSize)    // Size of the buffer
+Result ElfWriter<Elf>::ReadFromBuffer(const ElfReader<Elf>& reader)
```

Comment:
This function is not reading from a buffer.  It should have a different name.  Sorry I can't think of anything good now.  One possibility is `CopyFromReader`.

Reply:
Done.